### DISCUSSION TITLE
Beginner issue

### DISCUSSION THREAD

**Discussion by Boberthy-py** [phydrus/phydrus#22] [Q&A]
Hello!

I was working on a code and keep getting an error that the .OUT files can't be found? Is there something I did wrong in the set up of phydrus that they're not writing the .OUT files?

Thanks
- Boberthy

**Comment by mvremec** [comment]
Hi Boberthy!
Are the input files(SELECTOR.IN, T_LEVEL.OUT...) written in your working folder or is the folder empty?
Also, are you working on Windows/Linux/Mac?

**Comment by Boberthy-py** [comment]
Morning!

The PROFILE, ATMOSPH.IN and SELECTOR.IN are being written in my working folder but not the .OUT files. And I'm working on Windows

Thanks
-Boberthy

**Comment by baramousa** [comment]
Hi i have similar issue , only the profile, atmosphere in and secelctor are in the folders. i am using windows. where i run one of your examples in the cmd i get message that "libquadmath-0.dll" , "libwinpthread-1.dll" and "libgcc-s-seh-1.dll" were not found . 

Any suggestions ?

**Reply by raoulcollenteur** [reply]
You probably need to recompile the FORTRAN code, as some libraries are not available.
